src/app/containers/FauxHeadline/index.jsx

```diff
@@ -11,6 +14,14 @@ import { GridItemConstrainedLarge } from '#lib/styledGrid';
 // missing CSS 'display: block;' in psammead branch `explicitly-set-h1-styles-display-and-font-weight`
 const FauxHeadline = props => <Headline as="strong" {...props} />;
 
+const StyledFauxHeadline = styled(FauxHeadline)`
```

Comment:
seems odd to give something that is visually hidden spacing styles

Reply:
Just for future ref (in case we forget out conversation).

It appears `FauxHeadline` is `aria-hidden` but visible, while `VisuallyHiddenHeadline` is visually hidden, but aria visible.